bears/general/CPDBear.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
+    executable = 'run.sh'
+    arguments = "cpd --minimum-tokens 10" + \
+                " --files {filename}"
+    output_regex = re.compile(
+        r'(?P<message>.*) in')
+
+    def run(self, filename, file):
+        '''
+        Checks duplicate code with ``cpd``.
```

Comment:
It supports c++, java, C#, js, fortran, go, jsp, ObjectiveC, php, python, ruby and scala^^